### ISSUE TITLE
Dividing work for Summer Of Code

### ISSUE THREAD

**Issue by namangupta01** [publiclab/simple-data-grapher#14]
Hi @IshaGupta18, I think it would be better if we divide our work here and make a checklist so that we have a list of all the work to be done and also know who is working on which part? And also it will keep track of our work.
What do you say? and @jywarren also?

**Comment by IshaGupta18** [comment]
Yes I think this would be a wise thing to do. To do this, we should have a
look at our proposals and mix them up and divide the work such that we get
to do the best of all there is and we are able to learn new things from the
project as well as each other! Thanks!

**Comment by namangupta01** [comment]
Yes, let's divide the issues.
[] - Test Setup --> Naman
[] - Testing --> Naman and Isha
[] - Collection of Sample Data and Table Display --> Isha (Already Working)
[] - Plotting Graph Using Chart.js --> Naman (Already Working)
[] - Export Feature
Please feel free to add further work in the list.

**Comment by jywarren** [comment]
Cool, Isha how do you feel about this? Any suggested changes, additions?

Documentation of how to install/embed on another app, maybe...
https://github.com/publiclab/inline-markdown-editor/ has decent docs for
this, as does BlurredLocation.

UI flow?

**Comment by IshaGupta18** [comment]
I think this is okay for now, I will add some more checkpoints soon. We can divide the documentation work here and I'll more points about UI flow.

**Comment by IshaGupta18** [comment]
Hey @namangupta01 if your'e a little busy because of your exams right now, I could take on the chart.js task because I am little free right now. We can reshuffle the work again. What do you think?

**Comment by namangupta01** [comment]
Okay, so if you want to do it you can proceed. Thanks for so much
enthusiasm.

**Comment by IshaGupta18** [comment]
Oh thank you so much for such encouragement! I would definitely come to you
for suggestions in between! Thanks again.

**Comment by IshaGupta18** [comment]
Also, do have a look at #17 whenever you get the time. Me and @jywarren
have posted some good export ideas there, please give your views on them!
Thanks a lot!

**Comment by jywarren** [comment]
Also noting this blog as interesting, I forget it i shared already!

https://adamlynch.com/flexible-data-tables-with-css-grid/?1

**Comment by IshaGupta18** [comment]
Oh yes! I have read about this one! Thank you
